src/server/ads.ts

```diff
@@ -92,3 +92,23 @@ function parseAd(adJson: sqlAd): Advertisement {
         }
     };
 }
+
+export async function closeAd(adId: number): Promise<boolean> {
+    const answer: CloseOpenAdSQL = await fetch(`/ads/close/${adId}`, {
+        method: 'POST'
+    }).then(response => {
+        return response.json();
+    });
+
+    return answer.is_close;
+};
+
+export async function openAd(adId: number): Promise<boolean> {
+    const answer: CloseOpenAdSQL = await fetch(`/ads/open/${adId}`, {
```

Comment:
Again, both open and close functions looks the same, consider combining them together

Reply:
Done thanks